src/components/ColorPicker.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+
+import reactCSS from "reactcss";
+
+import { RGBColor, SketchPicker } from "react-color";
+
+interface ColorPickerProps {
+  regionId: string;
+  color: RGBColor;
+  onChange: (id: string, color: RGBColor) => void;
+}
+
+export default function ColorPicker(props: ColorPickerProps) {
+  const [display, setDisplay] = useState(false);
+
+  const styles = reactCSS({
```

Comment:
Dlaczego `reactCss`??
Jeśli dobrze pamiętam, to @sn3ak1  już zaczął wcześniej używać API od Material UI.
Według mnie lepiej tego nie mieszać i trzymać się jakiejś jednej biblioteki.


PS
Tak od siebie mogę polecić bibliotekę `[styled-components](https://styled-components.com/)`. Ona jest super pod tym względem, że składnia jest bardzo podobna do `css`. 
Oprócz tego do niej są ładne wtyczki do różnych IDE (napewno jest do webstrorm'a oraz vs code'a)
Oprócz tego, ona propsuje dobre praktykie, ponieważ teraz każda klasa jest per komponent, dzięki czemu można uniknąc sytuację w której zmieniamy coś w jednym miejscu, a potem zmienia się coś jeszcze gdzieś. Wiem, że bardzo niejaśnie to wytłumaczyłem :D. Jutro na spotkaniu powiem o co mi chodzi, tylko przypomnijcie o tym proszę.)